README.md

```diff
@@ -39,7 +39,7 @@ It's also possible to install `nixfmt` directly from the repository using `nix-e
 Also, updates are not done automatically (as it would with the system packages).
 
 ```console
-$ nix-env -f https://github.com/NixOS/nixfmt/archive/master.tar.gz -i
+$ nix-env -i -f https://github.com/NixOS/nixfmt/archive/master.tar.gz -i
```

Comment:
There's already a -i at the end :)

Reply:
Not on my screen:
<img width="916" alt="Screenshot 2025-03-10 at 16 15 53" src="https://github.com/user-attachments/assets/e8d35fc9-22f6-4449-a3b6-4f4ebf9f2eec" />

Reply:
It's fine to _move_ the `-i` to the beginning for clarity, but then you should also to remove the one at the end

Reply:
Done